src/bindings/jna/CMakeLists.txt

```diff
@@ -27,6 +27,21 @@ if (Java_VERSION_MAJOR GREATER 0 AND Java_VERSION_MINOR GREATER 7)
 		NAMES jna
 		PATHS /usr/share/java
 		)
+	
+	if(NOT jna) # try fetching it, remember to update SHA1 if you update the jar
+		file(DOWNLOAD 
+			"https://maven.java.net/content/repositories/releases/net/java/dev/jna/jna/4.2.2/jna-4.2.2.jar" 
```

Comment:
It is weird, https://cmake.org/cmake/help/v3.0/module/UseJava.html says that "For backwards compatibility, jar files listed as sources are ignored.", but it does not say anything about how to include them. INCLUDE_JARS only seems to set the include path, but not actually include them.

Actually we would only need to remove the line 451 of /usr/share/cmake-3.0/Modules/UseJava.cmake which does this strange ignoring of jar because of "backward compatibility".